src/main/java/com/spikes2212/path/NoLookaheadPointException.java

```diff
@@ -0,0 +1,12 @@
+package com.spikes2212.path;
+
+public class NoLookaheadPointException extends Exception {
```

Comment:
rename to `LookahedPointNotFoundException` or `LookahedPointOutOfBoundException`